src/OrchardCore.Modules/OrchardCore.Alias/Manifest.cs

```diff
@@ -6,6 +6,9 @@
     Website = "http://orchardproject.net",
     Version = "2.0.0",
     Description = "The alias module enables content items to have custom logical identifier.",
-    Dependencies = "OrchardCore.ContentTypes",
+    Dependencies = new string[]
```

Comment:
If there is only one dependency I'd suggest to format it this way:

```
    Dependencies = new [] { "OrchardCore.ContentTypes" },
```

Note that you can drop the `string` as it can be inferred

Reply:
Okay.